scripts/generate_reports.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+for var in "$@"
+do
+    	case $var in
+		--help)
+			echo '-j RUN JACOCO'
+			echo '-f RUN FINDBUGS'
+			echo '-c RUN CHECKSTYLE'
+			echo '--all RUN ALL PLUGINS'
+			;;
+		-j)
+			(cd ../ ; mvn install)
+			echo 'RUN JACOCO'
+			(cd ../ ; mvn jacoco:report)
+			(cd ../client/target/site/jacoco/ ; sensible-browser index.html)
```

Comment:
:-1: `sensible-browser` blocks (synchronous) and it may open a system-config browser instead of my preferred one. Also, there are talks of merging it into `xdg-open`. Why not `xdg-open` then?

See:

1. https://github.com/pkg/browser/commit/1111d28c951aa8f5816b080a48658dde7ab02bb8
2. https://github.com/flynn/flynn/issues/2125#issuecomment-156953583
3. https://bugs.launchpad.net/ubuntu/+source/gnome-control-center/+bug/562476
4. https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=539191